cityofaustin/atd-moped
# Use DataGridPro in place of normal DataGrid

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/16993

I switched the instances of DataGrid we have in Moped to DataGridPro so we can take advantage of the column resizing and so that the column menu looks the same for all DataGrid tables. 

- Phases Table
- Work Activity Table
- Staff Table

While I was upgrading the Staff Table, I also fixed a bug Amenity found where the sorting for Roles was not actually sorting on the Role name. (sidenote, there is a user in prod with a string in the role column, instead of an array of one string as the other ones have. So that user's role appears only as N. I tried to fix it by updating their profile in prod, but then discovered that we cant update non login users)

## Testing
**URL to test:** 

https://deploy-preview-1338--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

Check the Staff View table, sort by role. Try hiding and resizing columns. 
Phases table should operate the same as before, except for resizing columns. 
Work Activity table: try hiding and resizing columns. The column menus should look the same as the one in the project list table. 

Confirm that the background color of the pinned columns/header row matches the rest of the table. 

---
#### Ship list
